pkg/asset/templates/content/openshift/community-operators.go

```diff
@@ -0,0 +1,62 @@
+package openshift
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/templates/content"
+)
+
+const (
+	communityOperatorsFilename = "cluster-networkconfig-crd.yaml"
```

Comment:
is this the right filename? Wouldn't the community operators file be `community-operators.yaml `?

Reply:
Good catch, fixed